<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="1">
    <widget class="QTabWidget" name="tabWidget">
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>300</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>All_Data</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="1" rowspan="2">
        <widget class="QPushButton" name="All_Data_read_csv_Pushbutton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>출력하기</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>csv파일을 입력하시오.</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="All_Data_read_csv_lineEdit">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QTextEdit" name="All_Data_read_csv_TextEdit"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Predict</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QWidget" name="widget" native="true">
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="2" column="0">
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>수온</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QCheckBox" name="WaterTemp_checkBox">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="2">
             <widget class="QLineEdit" name="WaterTempLineEdit"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>인산염인</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QCheckBox" name="PhosphatePhosphorus_checkBox">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="3" column="0" colspan="2">
             <widget class="QLineEdit" name="PhosphatePhosphoruslineEdit"/>
            </item>
           </layout>
          </item>
          <item row="2" column="1">
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>염분</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QCheckBox" name="Salinity_checkBox">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="2">
             <widget class="QLineEdit" name="SalinitylineEdit"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>아질산질소</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QCheckBox" name="NitrousAcidNitrogen_checkBox">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="3" column="0" colspan="2">
             <widget class="QLineEdit" name="NitrousAcidNitrogenlineEdit"/>
            </item>
           </layout>
          </item>
          <item row="2" column="2">
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>용존산소</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QCheckBox" name="DissolvedOxygen_checkBox">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="0" column="2" rowspan="4">
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="0">
               <widget class="QPushButton" name="SearchpushButton">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="text">
                 <string>확인</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QPushButton" name="ResetpushButton">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="text">
                 <string>리셋</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>규산규소</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QCheckBox" name="SilicicAcidSilicon_checkBox">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="2" column="0" colspan="2">
               <widget class="QLineEdit" name="SilicicAcidSiliconlineEdit"/>
              </item>
             </layout>
            </item>
            <item row="1" column="0" colspan="2">
             <widget class="QLineEdit" name="DissolvedOxygenlineEdit"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>질산규소</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QCheckBox" name="NitricAcidNitrogen_checkBox">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="3" column="0" colspan="2">
             <widget class="QLineEdit" name="NitricAcidNitrogenlineEdit"/>
            </item>
           </layout>
          </item>
          <item row="3" column="0" colspan="3">
           <widget class="QTextEdit" name="ResultTextEdit"/>
          </item>
          <item row="0" column="0" colspan="3">
           <widget class="QLabel" name="label_3">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>          csv파일 입력 : </string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="3">
           <widget class="QLineEdit" name="input_csv_lineEdit"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Epilogue</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_9">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'휴먼명조'; font-size:14pt;&quot;&gt;정선해양관측자료를 통해본 유해적조&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'휴먼명조';&quot;&gt;김준한 · 강민석 · 이동현 &lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'휴먼명조';&quot;&gt;신라대학교 빅데이터연구실&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'휴먼명조'; font-size:11pt;&quot;&gt;The toxic red tide, ocean observation information through the Choice&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'휴먼명조';&quot;&gt;Kim-Jun Han · Gang-Min Seok · Lee-Dong Hyeon&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'휴먼명조';&quot;&gt;Korea Silla Univercity BigData Lab&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'휴먼명조';&quot;&gt;E-mail : axzswq@sillain.ac.kr&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'휴먼명조';&quot;&gt;적조현상은 우리의 생태계나 인류에게 위협적인 환경문제 중 하나이다. &lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'휴먼명조';&quot;&gt;그 중에서 유해적조는 정말로 치명적이다. 이러한 적조 원인은 플랑크톤의 이상 증식이다. &lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'휴먼명조';&quot;&gt;이상 증식에 대해서 연구는 많이 있지만, 정확히 어떤 성분이 어느 정도의 &lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'휴먼명조';&quot;&gt;원인을 제공하는지 자세히 나온 자료가 없다.&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'휴먼명조';&quot;&gt;본 논문에서 저자들은 해양요인들 중 정선해양관측정보에서 &lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'휴먼명조';&quot;&gt;수온, 염분, 용존산소, 인산염인, 아질산질소, 질산규소, 규산수소의 7개의 요소를 &lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'휴먼명조';&quot;&gt;python의 pandas 라이브러리를 이용하여서 데이터 추출, 과거적조발생자료를 &lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'휴먼명조';&quot;&gt;국립수산과학원에서 웹 크롤링한 후에 자료들과 통합하여서 &lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'휴먼명조';&quot;&gt;영향력을 제공하는 요소를 로지스틱 회귀분석 방식으로 분석했다. &lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'휴먼명조';&quot;&gt;2000~2017년 정보들을 대상으로 진행하였다.&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'휴먼명조';&quot;&gt;본 논문을 통해 해양에 적조가 생기는 이유를 정밀하게 발견하고 &lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'휴먼명조';&quot;&gt;적조발생을 예방하며 생태계와 학술계에 도움이 되기를 희망합니다.&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;담당 교수)윤홍원 교수님&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
